File: fibindex/fib.java
//RUN IT BY
//   javac fib.java;time java fib

public class fib {
    static int fibc(int n) {
      if (n < 3)
        return 1;
      return fibc(n - 1) + fibc(n - 2);
    }
    public static void main(String[] args) {
        System.out.println(fibc(40)); //31 - GNU
    }
}

File: fibindex/fib.erl
%RUN IT BY
%   erlc fib.erl; time echo 'fib:start().'|erl

-module(fib).
-export([start/0]).

fib(0) -> 0;
fib(1) -> 1;
fib(2) -> 1;
fib(N) -> fib(N - 1) + fib(N - 2).

start() -> fib(36).